src/Client/Operations/Parameters/ValuePattern.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace KeenIO\Client\Operations\Parameters;
+
+
+class ValuePattern
```

Comment:
All these classes should be removed. They don't belong to adding support for access keys, but to the refactoring that you reverted.

Reply:
removed